webpack/assets/javascripts/react_app/components/UserJwt/components/UserJwtForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Grid, Dropdown, MenuItem } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import { translate as __ } from '../../../common/I18n';
+
+const UserJwtForm = ({ handleSubmit }) => {
+  const expirationUnits = [
+    { label: __('Never'), value: 'never' },
+    { label: __('Hour'), value: 'hours' },
+    { label: __('Day'), value: 'days' },
+    { label: __('Week'), value: 'weeks' },
+    { label: __('Month'), value: 'months' },
+  ];
+
+  const [expirationUnit, setExpirationUnit] = useState(expirationUnits[0]);
+  const [expirationValue, setExpirationValue] = useState('');
+
+  const changeUnit = u => {
+    if (u.value === 'never') {
+      setExpirationValue('');
+    }
+    setExpirationUnit(u);
+  };
+
+  return (
+    <React.Fragment>
+      <Grid.Row style={{ marginBottom: '4px' }}>
+        <Grid.Col>
+          <div className="form-group">
+            <label className="control-label col-md-2">Expiration</label>
+            <div className="col-md-2">
+              <div className="input-group">
+                <input
+                  type="number"
+                  className="jwt-expiration-value form-control"
+                  min={1}
+                  disabled={expirationUnit.value === 'never'}
+                  value={expirationValue}
+                  placeholder={__('Expire in')}
+                  onChange={e => setExpirationValue(e.target.value)}
+                />
+                <div className="input-group-btn">
+                  <Dropdown id="expiration_unit">
+                    <Dropdown.Toggle>{expirationUnit.label}</Dropdown.Toggle>
+                    <Dropdown.Menu id="settings-dropdown">
+                      {expirationUnits.map(unit => (
+                        <MenuItem
+                          key={unit.value}
+                          onClick={() => changeUnit(unit)}
+                        >
+                          {unit.label}
+                        </MenuItem>
+                      ))}
+                    </Dropdown.Menu>
+                  </Dropdown>
+                </div>
+              </div>
+            </div>
+          </div>
+        </Grid.Col>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Col md={4}>
+          <input
+            type="button"
+            onClick={() => handleSubmit(expirationUnit, expirationValue)}
+            className="btn btn-success pull-right"
+            value={__('Generate')}
+          />
```

Comment:
Can this be success Button instead of input?
https://patternfly-react-pf3.surge.sh/?path=/story/patternfly-react-widgets-button--button